src/main/kotlin/no/nav/tpts/mottak/clients/saf/SafClient.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.tpts.mottak.clients.saf
+
+import io.ktor.client.request.header
+import io.ktor.client.request.post
+import io.ktor.features.NotFoundException
+import io.ktor.http.Url
+import kotlinx.coroutines.runBlocking
+import no.nav.tpts.mottak.clients.AzureOauthClient.getToken
+import no.nav.tpts.mottak.clients.HttpClient.client
+import no.nav.tpts.mottak.graphql.Graphql
+import no.nav.tpts.mottak.graphql.JournalfortDokumentMetaData
+import no.nav.tpts.mottak.graphql.SafQuery
+import no.nav.tpts.mottak.graphql.SafQuery.Variantformat.ORIGINAL
+import no.nav.tpts.mottak.graphql.journalpost
+
+const val SAF_URL = "https://saf.dev-fss-pub.nais.io"
+
+object SafClient {
+    private val token = runBlocking { getToken().accessToken }
+
+    suspend fun hentMetadataForJournalpost(journalpostId: String): JournalfortDokumentMetaData {
+        val safResponse: SafQuery.Response = client.post(url = Url("$SAF_URL/graphql")) {
+            header("Authorization", "(Bearer $token")
```

Comment:
for header names (Authorization, Accept, Content-Type) og verdier (application/json) kan du gjerne bruke konstanter, f.eks. de som finnes i `HttpHeaderNames` eller annen egnet ktor-klasse